packages/loader/container-loader/src/containerContext.ts

```diff
@@ -169,6 +169,12 @@ export class ContainerContext implements IContainerContext {
 
     private readonly _fluidModuleP: Promise<IFluidModuleWithDetails>;
 
+    /**
+     * {@inheritDoc @fluidframework/container-definitions#IContainerContext.entryPoint}
+     */
+    public readonly entryPoint?: Promise<FluidObject | undefined>;
```

Comment:
i think a getter would be better:
```
public get entryPoint(): undefined | Promise<FluidObject | undefined>{
    return this._deferredEntryPoint.promise;
}
```

Reply:
i'm fine with the other approach called out above. neither is really great, and just and an area that needs work overall.